src/sonic-build-hooks/scripts/buildinfo_base.sh

```diff
@@ -151,7 +151,7 @@ set_reproducible_mirrors()
 download_packages()
 {
     local parameters=("$@")
-    local filenames=
+    unset filenames
```

Comment:
Looks like unset filenames is not necessary, because the line 155 declare without -g options.